### PR TITLE
Feature/remove license from UI

### PR DESCRIPTION
**Pull request by guilpier-code**
This PR removes all elements related to license in the GUI : 
- The window associated to the connexion to the metrics / license server
- In the GUI's help menu, the items "Continue on/off line"
- The message when using Antares for the first time on a machine as well as the consent/refusal to stay online during the session (this information was stored in a **antares.hwb** somewhere on the machine - in AppData for Windows). This 2 elements were tightly dependent, so they were removed together.

### CODE REVIEW

File: src/ui/simulator/application/main/main.cpp [hunk 1]
```diff
@@ -181,8 +181,8 @@ EVT_MENU(mnIDHelpPDFOptimizationProblemsFormulation,
 EVT_MENU(mnIDHelpPDFSystemMapEditorReferenceGuide,
          ApplWnd::evtOnHelpPDFSystemMapEditorReferenceGuide)
 EVT_MENU(mnIDHelpPDFExamplesLibrary, ApplWnd::evtOnHelpPDFExamplesLibrary)
-EVT_MENU(mnIDHelpContinueOnline, ApplWnd::evtOnHelpContinueOnline)
-EVT_MENU(mnIDHelpContinueOffline, ApplWnd::evtOnHelpContinueOffline)
+// EVT_MENU(mnIDHelpContinueOnline, ApplWnd::evtOnHelpContinueOnline)
+// EVT_MENU(mnIDHelpContinueOffline, ApplWnd::evtOnHelpContinueOffline)
```
Comment (flomnes) [inline src/ui/simulator/application/main/main.cpp]: Remove ?

Comment (guilpier-code) [reply src/ui/simulator/application/main/main.cpp]: now corrected

**Review by flomnes — CHANGES_REQUESTED** [review]
You may remove this declaration

```cpp
    void evtOnShowID(wxCommandEvent&);
```

from **src/ui/simulator/application/main/main.h** since this function is no longer called.

**Comment by guilpier-code** [comment]
you're right, the correction was brought

**Review by flomnes — APPROVED** [review]

**Pull request merged**